### ISSUE TITLE
Success Criterion 2.2.2 - Pause, Stop, Hide - Level A

### ISSUE THREAD

**Issue by JJdeGroot** [w3c/matf#33]
WCAG2ICT guidance: https://www.w3.org/TR/wcag2ict-22/#pause-stop-hide

Share your thoughts for applying to mobile apps as a comment below.

**Comment by qbalsdon** [comment]
Does the mechanism have to exist within app or can we just say that it should at least respect system settings?

**Comment by JJdeGroot** [comment]
<details>
<summary>MATF meeting on July 31, 2024</summary>

[Source](https://www.w3.org/2024/07/31-matf-minutes.html#t05)

So many failures

<julianmka> I have to drop for a work meeting. Thanks, y'all!

adb control for ttr Joe_Humbert: settings put secure accessibility_interactive_ui_timeout_ms [MILLIS] % millis are additive - they add extra time

<Joe_Humbert> thankx Quintin

I think there should be guidance given the dominance of video on mobile - it should stay paused if the same element type is paused

<Mick> +1 quintinb

Either every element should be pause-able or at least respect settings

Joe_Humbert: Just to raise: Android and iOS treat it differently when they refer to the setting - I wonder if iOS (reduce animations) is actually satisfactory

<quintinb> +1 Joe - is that actually stop animations?

I hate that terminaology - so vague

GleidsonRamos should we save the user preference in app?

<quintinb> +1 GleidsonRamos yes I think there should be guidance on that

<Joe_Humbert> great question

GleidsonRamos should we sync app setting with OS setting?

<quintinb> +1 yes I think that is the way

JJ there is probably some guidance on this type of situation

Jamie: Just to clarify: there is a feature for reduce motion and auto play animated images - seems like apple has 2 settings

<Joe_Humbert> +1 jamie. maybe I was confused

We should probably be clear about what we want and let the OS sort themselves out

JJ any new agenda items?

JJ how do we deal with OS vs / system settings

quintinb: maybe we should just focus on the desired behaviour

Joe_Humbert: We should honour users system settings

<JJ> +1

<quintinb> +1 Joe_Humbert
</details>

**Summary:**

**quintinb** suggests guidance for pausing video on mobile and respecting system settings for all elements. **Mick** agrees.

**quintinb** and **Joe_Humbert** discuss the differences between Android and iOS settings for reducing animations, expressing concerns over vague terminology. **Jamie** joins the discussion.

**GleidsonRamos** asks if user preferences should be saved in the app and synced with OS settings. **quintinb** agrees and thinks guidance is needed.

**Jamie** clarifies features for reducing motion and autoplay on Apple devices. **quintinb** agrees on the need for clear guidance and letting the OS handle specifics.

**quintinb** asks if there are new agenda items and how to handle OS vs. system settings.

**Joe_Humbert** emphasizes the importance of honoring user system settings. **JJ** and **quintinb** agree.

**Comment by julianmka** [comment]
Based on previous task force conversation, this SC can be applied to native mobile apps and mobile web with minimal or no deviation from WCAG2ICT.

Proposal
In MATF's first draft of guidance, this SC's section will [read as presented in WCAG2ICT](https://www.w3.org/TR/wcag2ict-22/#pause-stop-hide) except for wording adjustments that reflect native mobile app and mobile web context.

Please indicate your agreement with a thumbs up emoji 👍, or if you disagree, use the thumbs down emoji 👎 and elaborate in comments.

**Comment by JJdeGroot** [comment]
A few notes/comments below from e-mail conversation with @alastc (AC)

**JJ:**
For 2.2.2 Pause, Stop, Hide, developers can also use system settings such as reduce motion to stop or hide motion.
- But... on iOS the setting is named "reduce motion", when motion is stopped or hidden, it's not "reduced" as the settings implies
- And users might not be aware of these settings, it's also not always possible to deeplink to the relevant settings screen.

**AC:**
I think in general we have assumed that a widely-available user-setting can be the mechanism. If having that setting on then passed the criteria text the content (/app) would pass.

**JJ**: 
Okay, the question is when is a setting considered widely-available? Would auditors need to write this down in the "Accessibility support baseline" section? E.g. some settings are only available on newer Android version, and some settings be buried deep in the settings screens.

In WCAG 2.x we’ve taken that on a case by case basis but, for example, a system setting is sufficient to pass Animation from interactions in [C39](https://www.w3.org/WAI/WCAG22/Techniques/css/C39). This sounds similar.

**AC:**
This topic is a constant discussion, there isn’t a clear and concrete answer for a world-wide standard. However, if the setting is free (once the base device and OS are paid for), and on more than 66% of used devices it should be a fairly safe option.

**Comment by detlevhfischer** [comment]
From the discussion of this in the web context it's worth noting that in some UA, "prefers-reduced-motion" is not even in the settings - it seems in Firefox you need to delve into `about:config`, pass some warnings and find the flag before it is set they way you want it. Granted, in recent versions of iOS and Android it is more straightforward. Would still be interesting to research empirically how many of the users negatively affected by motion will 
- A. Know there is an accessibility setting for that 
- B. Successfully locate that setting.

**Comment by julianmka** [comment]
After discussion and voting at the meeting today, the task force members present accepted the [proposed language](https://github.com/w3c/matf/issues/33#issuecomment-2313970437) for the first draft of our guidance.

**Comment by JJdeGroot** [comment]
Closing this issue because the draft language has been accepted.

**Comment by jamieherrera** [comment]
Just in case we come back to this SC in an updated 2.0 version after 2025, we should add a best practice note for the dev to honor a user's OS Accessibility motion settings by including the appropriate Boolean-enabled setting for reduced motion in the app; it's not a WCAG requirement since the SC has this 5 second minimum, and the user has to toggle the setting in the OS, but at least the dev's code shouldn't be the barrier. 
The WCAG2ICT notes sort of soften the 5 second expectation, maybe our 2.0 version could have similar language as a note, knowing that the WCAG language is... what it is.
